src/test/clip.t.sol

```diff
@@ -801,4 +801,80 @@ contract ClipperTest is DSTest {
 
         dog.bark(ilk, me);
     }
+
+    // At a stopped == 1 we are ok to take
+    function test_stopped_take() public takeSetup {
+        clip.setBreaker(1);
+        // Bid so owe (= 25 * 5 = 125 RAD) > tab (= 110 RAD)
+        // Readjusts slice to be tab/top = 25
+        Guy(ali).take({
+            id:  1,
+            amt: 25 ether,
+            max: ray(5 ether),
+            who: address(ali),
+            data: ""
+        });
+    }
+
+    function testFail_stopped_take() public takeSetup {
+        clip.setBreaker(2);
+        // Bid so owe (= 25 * 5 = 125 RAD) > tab (= 110 RAD)
+        // Readjusts slice to be tab/top = 25
+        Guy(ali).take({
+            id:  1,
+            amt: 25 ether,
+            max: ray(5 ether),
+            who: address(ali),
+            data: ""
+        });
+    }
+
+    function test_stopped_auction_reset_tail() public {
+        auctionResetSetup(10 hours); // 10 hours till zero is reached (used to test tail) 
+
+        clip.setBreaker(1);
+
+        pip.poke(bytes32(uint256(3 ether))); // Spot = $1.50 (update price before reset is called)
+
+        (,,,, uint96 ticBefore, uint256 topBefore) = clip.sales(1);
+        assertEq(uint256(ticBefore), startTime);
+        assertEq(topBefore, ray(5 ether)); // $4 spot + 25% buffer = $5 (wasn't affected by poke)
+        
+        hevm.warp(startTime + 3600 seconds);
+        assertTrue(!try_redo(1));
+        hevm.warp(startTime + 3601 seconds);
+        assertTrue( try_redo(1));
+        
+        (,,,, uint96 ticAfter, uint256 topAfter) = clip.sales(1);
+        assertEq(uint256(ticAfter), startTime + 3601 seconds);     // (now)
+        assertEq(topAfter, ray(3.75 ether)); // $3 spot + 25% buffer = $5 (used most recent OSM price)
+    }
+
+    function testFail_stopped_auction_reset_tail() public {
+        clip.setBreaker(2);
+
+        auctionResetSetup(10 hours); // 10 hours till zero is reached (used to test tail) 
+
+        pip.poke(bytes32(uint256(3 ether))); // Spot = $1.50 (update price before reset is called)
+
+        (,,,, uint96 ticBefore, uint256 topBefore) = clip.sales(1);
+        assertEq(uint256(ticBefore), startTime);
+        assertEq(topBefore, ray(5 ether)); // $4 spot + 25% buffer = $5 (wasn't affected by poke)
+        
+        hevm.warp(startTime + 3600 seconds);
+        assertTrue(!try_redo(1));
+        hevm.warp(startTime + 3601 seconds);
+        assertTrue( try_redo(1));
+    }
+
+    // does nothing but we can still test the breaker works
+    function test_yank() public {
+        clip.setBreaker(3);
+        clip.yank();
```

Comment:
As I mentioned on your other PR, `yank` shouldn't require the circuit breaker to be turned on. Or, at least there should be a version of it that is just `auth` for the `End` to call. You can, for example, make a wrapper around it and have that require the breaker to be on (so long as you also `rely` the contract on itself). Or just have two functions.

Reply:
Ah yeah my bad forgot about this